libs/sdk-model/src/objRef/factory.ts

```diff
@@ -34,3 +34,14 @@ export function uriRef(uri: Uri): UriRef {
 export function localIdRef(localIdentifier: Identifier): LocalIdRef {
     return { localIdentifier };
 }
+
+/**
+ * Creates an ObjRef from a id and uri
+ * @param uri - URI to use
+ * @param identifier - identifier to use
+ * @returns obj ref reference
+ * @public
+ */
+export function objRef(uri: string, identifier: string): ObjRef {
```

Comment:
I am not sure we want to promote this "pattern". At least we should make this internal or something.
@kandl WDYT?

Reply:
Moved to dashboard package.